### DISCUSSION TITLE
Screen Position issue

### DISCUSSION THREAD

**Discussion by adeniyisikirat3** [EmulatorJS/EmulatorJS#461] [General]
I have only tried on the Gameboy advance(I can't say if this Same issue is on the other consoles / emulators). 

So when playing on the game boy advance the screen hangs to low & close to the virtual game-pad even when allow full-screen, it's a bit distracting Because of the virtual pad buttons.   is there a way to adjust ?  Maybe move it up to the top  ?

[Edit: I'm playing on Chrome browser with an Android phone ]

**Comment by adeniyisikirat3** [comment]
is there a way I could get the older versions  of the emulator through https://emulatorjs.org/editor.html   ?

Older versions like : 
Emulator js  usingVersion='0.4.23-05'



That is Version='0.4.23-05

The one with the red analogue joy stick &  " X,Y,A,B " buttons,  cos that's the version i saw first that prompted me to come here. That one was perfect & The screen position was perfect too.


Is there a way I cld get older versions ?

**Comment by ethanaobrien** [comment]
No, that is not this project and was never an old version. Please do some digging. I have an entire site with documentation. Please see this for how to edit the gamepad settings: https://emulatorjs.org/docs4devs/Virutal%20Gamepad%20Settings.html

**Comment by adeniyisikirat3** [comment]
I am so sorry Boss , but that was a prediction text error 

I meant to type : " https://emulatorjs.org/editor.html " instead

Forgive my carelessness

,
Ok  I would look at how to edit the gamepad settings

**Comment by adeniyisikirat3** [comment]
🙋🏾‍♀ Hello Ethana, good day & hope you're well.

I have looked at it and here is my feedback:


`<div style="width:640px;height:480px;max-width:100%">
            <div id="game"></div>
        </div>
        <script>
            EJS_player = "#game";
            EJS_core = "gba";
            EJS_gameName = "Game's-Name";
            EJS_AdUrl = "ht.tps://www.¿¿ EXAMPLE Advert-link¿¿.com/";
            EJS_color = "";
            EJS_startOnLoaded = false;
            EJS_VirtualGamepadSettings = [
    {
        type: "button",
        text: "Y",
        id: "y",
        location: "right",
        left: 40,
        bold: true,
        input_value: 9
    },
    {
        type: "button",
        text: "X",
        id: "X",
        location: "right",
        top: 40,
        bold: true,
        input_value: 1
    },
    {
        type: "button",
        text: "B",
        id: "b",
        location: "right",
        left: 81,
        top: 40,
        bold: true,
        input_value: 8
    },
    {
        type: "button",
        text: "A",
        id: "a",
        location: "right",
        left: 40,
        top: 80,
        bold: true,
        input_value: 0
    },
    {
        type: "zone",
        location: "left",
        left: "50%",
        top: "50%",
        joystickInput: true,
        inputValues: [16, 17, 18, 19]
    },
    
]
            EJS_pathtodata = "ht.tps://¿¿ EXAMPLE script-src-website¿¿/data/";
            EJS_gameUrl = "ht.tps://www¿¿ EXAMPLE rom-website¿¿.com/ROM-file.zip";
        </script>
        <script src="ht.tps://¿¿ EXAMPLE script-src-website¿¿/data/loader.js"></script>`






The above is what I tried. (view here: 'https://pastebin.com/ecJAAzXN' incase the one above is jumbled up.)


I tried that but it went blank

I Also tried " EJS_Buttons "
That one worked well 

but " EJS_VirtualGamepadSettings "
Makes it go blank

**Reply by ethanaobrien** [reply]
Is there anything in the console? If it's blank, that's a configuration error

**Reply by adeniyisikirat3** [reply]
> Is there anything in the console? If it's blank, that's a configuration error

Console ? 

What I did is : i just pasted that code in my html editor and see the output on the page's front-end

It works  fine ..only when I include " EJS_VirtualGamepadSettings " that's when it goes blank 

If you're asking if it's linked to a ROM then yes and the ROM plays & works fine  untill I include " EJS_VirtualGamepadSettings "  then it goes blank

Consider this suggestion  " https://github.com/EmulatorJS/EmulatorJS/discussions/464 " , 

For sake of simplicity so even a 5-year old could understand and quickly set up.

But if it would be too much work / stress , then leave it be ..I can manage it the way it is. It's still cool as it is nonetheless 👍

**Reply by ethanaobrien** [reply]
It may be done

> Console ?

Based on the fact you don't know what the console is I'm going to guess you're not actually a developer, and just trying to make money off ads or something. You need to right click, click "inspect", click the "console" tab in the window that pops up, right click, click "save", upload the file to GitHub

**Reply by adeniyisikirat3** [reply]
> It may be done
> 
> > Console ?
> 
> Based on the fact you don't know what the console is I'm going to guess you're not actually a developer, and just trying to make money off ads or something. You need to right click, click "inspect", click the "console" tab in the window that pops up, right click, click "save", upload the file to GitHub

Thank you very much boss 🙏🙇🏽‍♀ , I'd try that & give you a feedback.

& Yes, you're correct. I don't have experience in developing. But I have a small anime group I'm trying to grow.

I feel these games would be cool too keep my members entertained (I may be wrong). 
I like gba, I used to have a game-boy advance console back in school,  I lent to friend over the holidays. he moved & never returned it.  I played & enjoyed games:  final fantsy, Zeldda etc.  

I was looking up stuff on final fantsy dwn of souls when I came across your emulator on a website & played online
